expression/builtin_cast_vec_test.go

```diff
@@ -136,10 +137,11 @@ func (g *datetimeJSONGener) gen() interface{} {
 	minute := rand.Intn(60)
 	second := rand.Intn(60)
 	microsecond := rand.Intn(1000000)
-	d := types.Time{
-		Time: types.FromDate(year, month, day, hour, minute, second, microsecond),
-		Fsp:  3,
-	}
+	d := types.NewTime(
+		types.FromDate(year, month, day, hour, minute, second, microsecond),
+		mysql.TypeDatetime,
```

Comment:
0?

Reply:
guess datetime is better from the context.